docs/docs/components/pressable.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: pressable
+title: Pressable
+sidebar_label: Pressable
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import GifGallery from '@site/components/GifGallery';
+
+<GifGallery>
+  <img src={useBaseUrl('gifs/pressable.gif')} width="70%" />
+</GifGallery>
+
+:::info
+This component is a drop-in replacement for the `Pressable` component.
+:::
+
+`Pressable` is a component that can detect various stages of tap, press and hover interactions on any of its defined children.
```

Comment:
```suggestion
`Pressable` is a component that can detect various stages of tap, press and hover interactions on any of its children.
```

I know it's the same as in the react native docs, but I don't get the `defined` part. WDYT?

Reply:
I agree, it's not really necessary. @m-bert wdyt?

Reply:
Let's remove that 😉